packages/webpack-config-terra/package.json

```
@@ -48,11 +48,11 @@
     "glob": "^7.1.1",
     "memory-fs": "^0.4.1",
     "mini-css-extract-plugin": "^1.3.1",
-    "node-sass": "^5.0.0",
     "postcss-assets-webpack-plugin": "^4.0.0",
     "postcss-custom-properties": "^10.0.0",
```

while we're here, I think updating to the new version of postcss-custom-properites is passive.
```suggestion
    "postcss-custom-properties": "^11.0.0",
```